src/ServicePulse.Host/vue/src/composables/serviceServiceControl.js

```diff
@@ -79,9 +79,9 @@ export const connections = reactive({
 });
 
 export function useServiceControl() {
-  onMounted(() => {
-    useServiceControlStats();
-    useServiceControlMonitoringStats();
+  onMounted(async () => {
+    await useServiceControlStats();
+    await useServiceControlMonitoringStats();
```

Comment:
These should potentially be done in parallel with a `Promise.all`

Reply:
definitely, if equivalent behaviour is desired